sdk/keyvault/azure-keyvault-secrets/pyrightconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "reportMissingImports": false,
+  "pythonVersion": "3.10",
```

Comment:
We try to stick to running on the minimum supported version for the repo: https://github.com/Azure/azure-sdk-for-python/blob/main/pyrightconfig.json

Can you update to 3.8?